conf/zeppelin-site.xml.template

```diff
@@ -53,12 +53,30 @@
 </property>
 
 <property>
+  <name>zeppelin.bucket.name</name>
+  <value>zeppelin</value>
+  <description>path or URI for notebook persist</description>
+</property>
+
+<property>
+  <name>zeppelin.bucket.user</name>
+  <value>user</value>
+  <description>path or URI for notebook persist</description>
+</property>
+
+<property>
   <name>zeppelin.notebook.storage</name>
   <value>org.apache.zeppelin.notebook.repo.VFSNotebookRepo</value>
   <description>notebook persistence layer implementation</description>
 </property>
 
 <property>
+  <name>zeppelin.notebook.storage</name>
+  <value>org.apache.zeppelin.notebook.repo.S3NotebookRepo</value>
+  <description>notebook persistence layer implementation</description>
+</property>
```

Comment:
I think it's better not have two entries for the `zeppelin.notebook.storage` property at the same time.
How about put all S3NotebookRepo related configurations inside of single commented block `<!-- ... -->`  with some description.

Then user can still use this template without any modification, also user can easily understand what is necessary to setup S3NotebookRepo.